src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (value: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pageCount = Math.ceil(total / perPage);
+  const pageButtons = getNumbers(1, pageCount);
+  const firstPage = currentPage === 1;
+  const lastPage = currentPage === pageCount;
```

Comment:
Rename variables to show that it's boolean 
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig